models/userModel.js

```diff
@@ -1,11 +1,15 @@
-//connect through mongoose 
-
-//user schema - Users can create an account and log in.
-
-//declare a new mongoose instance
-//model user accounts
-
-//email/username
-//password
-
-//export
\ No newline at end of file
+const mongoose=require('mongoose');
+
+const userSchema=mongoose.Schema({
+    _id: mongoose.Schema.Types.ObjectId,
+    first_name: {type: String, required: true},
+    last_name: {type: String, required: true},
+    email: {type: String, 
+        required: true, 
+        unique: true,
+        match: /^[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/
+    },
+    password: {type: String, required: true}
+});
+
+module.exports=mongoose.model('User', userSchema);
```

Comment:
Add a new line this file @Lemi-Agrey

Reply:
I have not understood. you mean i new empty line